src/integrationTest/java/uk/gov/hmcts/reform/cwrdapi/CaseWorkerCreateUserWithFileUploadTest.java

```diff
@@ -115,16 +124,17 @@ public void shouldUploadServiceRoleMappingsXlsxFileSuccessfully_with_extra_space
     @Test
     public void shouldReturn200PartialSuccessWhenNameIsLongerThan128AndNameHasInvalidCharacter() throws IOException {
         Map<String, Object> response = uploadCaseWorkerFile(
-                "Staff Data Upload With Name Longer Than 128 and Name With Invalid Character.xlsx",
-                CaseWorkerConstants.TYPE_XLS, "200 OK", cwdAdmin);
+            "Staff Data Upload With Name Longer Than 128 and Name With Invalid Character.xlsx",
+            CaseWorkerConstants.TYPE_XLS, "200 OK", cwdAdmin);
 
-        assertThat(response.get("message")).isEqualTo(REQUEST_FAILED_FILE_UPLOAD_JSR);
-        assertThat(response.get("message_details"))
-                .isEqualTo(String.format("2 record(s) failed validation and 1 record(s) uploaded", 2));
+        assertThat(response).contains(entry("message", REQUEST_FAILED_FILE_UPLOAD_JSR))
+            .contains(entry("message_details",
+                format("2 record(s) failed validation and 1 record(s) uploaded", 2))).hasSize(6);
         assertThat(response.get("error_details").toString()).contains(FIRST_NAME_INVALID);
         List<CaseWorkerAudit> caseWorkerAudits = caseWorkerAuditRepository.findAll();
         assertThat(caseWorkerAudits.size()).isEqualTo(1);
-        assertThat(caseWorkerAudits.get(0).getStatus()).isEqualTo(PARTIAL_SUCCESS.getStatus());
+        assertThat(caseWorkerAudits.get(0).getStatus()).isEqualTo(PARTIAL_SUCCESS.getStatus()).hasSize(15);
+        ;
```

Comment:
Might just be GitHub's formatting but it looks like there is an additional `;` here

Reply:
done